_posts/2023-11-09-in-progress-wildfly-feature-on-openshift.adoc

```diff
@@ -0,0 +1,276 @@
+---
+layout: post
+title: 'Trying Out An In-progress Feature for WildFly on OpenShift'
+date: 2023-11-09
+tags: oidc authentication OpenShift scopes
+synopsis: How to deploy a WildFly app secured with OIDC on OpenShift using an in-progress feature. 
```

Comment:
same here, I would briefly specify what the in-progres feature is

Reply:
Fixed!